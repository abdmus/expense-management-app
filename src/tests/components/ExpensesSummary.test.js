import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={12} expensesTotal={333235} />);
    expect(wrapper).toMatchSnapshot();
});