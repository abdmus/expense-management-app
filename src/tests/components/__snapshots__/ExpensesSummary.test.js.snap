// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $2.35
     
  </h1>
</div>
`;

exports[`should correctly render with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $3,332.35
     
  </h1>
</div>
`;
